File: C++/14. Longest Common Prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        string str1=strs[0];
        string res;
        for (int i=0; i<str1.size(); i++){
            for (int j=1; j<strs.size(); j++){
                if (strs[j][i] != str1[i]){
                    return res;
                }
            }
            res+=str1[i];
        }
        return res;
    }
};

File: C#/14. Longest Common Prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        Array.Sort(strs);
        int n=strs.Length;
        string str1=strs[0];
        string str2=strs[n-1];
        string res="";
        for(int i=0;i<str1.Length;i++){
            if(str1[i]==str2[i]){
                res+=str1[i];
            }
            else{
                break;
            }
        }
        return res;
    }
}
